lib/trino-hive-formats/src/main/java/io/trino/hive/formats/line/json/JsonDeserializer.java

```diff
@@ -398,7 +402,7 @@ public DoubleDecoder()
         void decodeValue(JsonParser parser, BlockBuilder builder)
                 throws IOException
         {
-            DOUBLE.writeDouble(builder, parser.getDoubleValue());
+            writeDouble(builder, parser.getDoubleValue());
```

Comment:
Missing try / catch here for `JsonProcessingException` thrown by `parser.getDoubleValue()` ?